test/unittests/BreadcrumbTrailTest.php

```diff
@@ -44,21 +44,35 @@ protected function setUp(): void
     /**
      * Test __toString() returns correct string
      * TODO: Add potential edge cases (such as white space)
+<<<<<<< 334ece634082b9b39992f74814a5fb21fa9ecf51
```

Comment:
These files have confilcts, you need to fix them.

Reply:
it is resolved.